luaui/Widgets/cmd_commandq_manager.lua

```diff
@@ -0,0 +1,55 @@
+function widget:GetInfo()
+    return {
+        name = "Command Queue Manager",
+        desc = "Skips current command or cancels the last command in command",
+        author = "[DE]LSR",
+        date = "5 Apr, 2022",
+        license = "GNU GPL, v2 or later",
+        layer = 1, --  after the normal widgets
+        enabled = true --  loaded by default?
+    }
+end
+
+-- Handlers
+function widget:Initialize()
+    widgetHandler:AddAction("command_skip_current", SkipCurrentCommand)
```

Comment:
I recommend making it something like:

```
commandqueue skip
commandqueue cancel_last
```

This way all custom commands defined by this widget are scoped by a main command and subscoped by subcommands

Reply:
Or even something like:

```
commandqueue skip first
commandqueue skip last
```

Reply:
was determined with Beherith, that single command is likely ok, and there is no need to split up args